smf/cockroachdb/manifest.xml

```diff
@@ -4,7 +4,8 @@
 <service_bundle type='manifest' name='cockroachdb'>
 
   <service name='system/illumos/cockroachdb' type='service' version='1'>
```

Comment:
We should take this opportunity to rename this service to `oxide/cockroachdb`, I think.

Reply:
Done